protostar/testing/testing_summary.py

```diff
@@ -71,6 +72,20 @@ def log(
             log_color_provider.bold("Seed: ".ljust(header_width))
             + str(self.testing_seed)
         )
+        estimated_fees: dict[str, float] = {}
+        for result in self.test_results:
+            if (
+                isinstance(result, (PassedTestCaseResult, PassedFuzzTestCaseResult))
+                and result.estimated_fee
+            ):
+                estimated_fees[result.test_case_name] = result.estimated_fee
+        if estimated_fees:
+            logger.info(log_color_provider.bold("Estimated fees:".ljust(header_width)))
+            for test_name, estimated_fee in estimated_fees.items():
+                logger.info(
+                    f'    - for a single run of the test "{test_name}": '
+                    f"{estimated_fee} gas ({estimated_fee * 1_000_000_000} WEI)"
```

Comment:
i think that the result of that should be output in gas only, gas prices can vary.

Reply:
fixed in [new approach](https://github.com/software-mansion/protostar/pull/1013/commits/3efa247b9c0b19218279da5b017f02f38aff4ebf)